### PR TITLE
[eth] Complete syncPythState.js

### PR DESCRIPTION
**Pull request by ali-bahjati**
This PR adds the following checks to `syncPythState.js`:
- The contract should have no owner.
- The contract wormhole address and chain id should be correct.
- Update fee should be in sync. Updates it if not.
- Valid time period should be in sync. Updates it if not.
- Data sources should be in sync. Updates it if not.
- Governance data source should be in sync. If not, this script will throw an Error as currently (due to multisig cli design) it's not possible to sign VAAs from two data sources. Once we need to make this change we can update the related codes to it.

Note: This code is temporary and will be replaced by the automation efforts that some people are working on.

### CODE REVIEW

**Review by jayantk — APPROVED** [review]
very cool. 

one thing to consider is where this will plug in to the workflow automation. i think you simply output the update txs and feed them into @guibescos 's multisig script, but good to be explicit.

**Review by njkumr — APPROVED** [review]

File: ethereum/scripts/syncPythState.js [hunk 1]
```diff
@@ -150,43 +216,185 @@ async function upgradeContract(proxy) {
...
+    const setDataSourcesPayload = new governance.SetDataSourcesInstruction(
+      governance.CHAINS.unset,
+      Array.from(desiredDataSources).map(
+        (ds) =>
+          new governance.DataSource(
+            Number(ds[0]),
+            new governance.HexString32Bytes(ds[1])
+          )
+      )
+    ).serialize();
```
[the remark this answers is not in the document]
Comment (ali-bahjati) [reply ethereum/scripts/syncPythState.js:345]: Yeah you are right. We can revisit it later if we really require this to happen.

File: ethereum/scripts/syncPythState.js [hunk 2]
```diff
@@ -76,57 +78,121 @@ async function executeMultisigTxAndGetVaa(txKey) {
...
+
+  console.log(`Executing the VAA...`);
+  await proxy.executeGovernanceInstruction("0x" + vaa);
+
+  if (keepVaaCache !== true) {
+    cleanUpVaaCache(payload);
+  }
+}
+
+async function enesureWormholeAddrAndChainIdIsCorrect(proxy) {
```
[the remark this answers is not in the document]
Comment (ali-bahjati) [reply ethereum/scripts/syncPythState.js:160]: Thanks.

File: ethereum/scripts/syncPythState.js [hunk 3]
```diff
@@ -1,9 +1,11 @@
 const governance = require("@pythnetwork/xc-governance-sdk");
```
[the remark this answers is not in the document]
Comment (ali-bahjati) [reply ethereum/scripts/syncPythState.js:1]: Yes will do.

**Pull request merged**